nums/numpy/api.py

```diff
@@ -261,6 +261,45 @@ def logspace(start, stop, num=50, endpoint=True, base=10.0, dtype=None, axis=0):
         ba = ba.astype(dtype)
     return ba
 
+############################################
+# Matrix Ops
+############################################
+
+def tensordot(x1: BlockArray, x2: BlockArray,
```

Comment:
Didn't we already merge this and `matmul`?

Reply:
I also wonder why it appears in the diff, since I did not do anything here.

Reply:
Have you merged the latest `api` branch into your branch? If you have, you may have merged in a way where these changes are marked as more recent. If you've merged and this is still happening, an easy way to correct the issue is to create a new branch from `api` and copy (or cherry pick) your changes over. You can also go back to the point at which `api` was merged into your branch and correct the merge, but this is a more complicated, time consuming, and error prone approach.

Reply:
That was repetitive code due to the merge error, I guess. I removed one of them.

Reply:
This is still an issue. I'm okay with you creating a new branch from `api`, updating it with changes from this branch, and creating a new PR.